src/components/DataTable/DataTable.scss

```diff
@@ -162,7 +162,7 @@ $breakpoint: 768px;
 
 .Footer {
   padding: spacing();
-  background: color('sky', 'light');
+  background: var(--p-surface-foreground, color('sky', 'lighter'));
```

Comment:
Likewise this should be a subdued foreground. I'm also going to update the text color so the contrast is correct.